framework-plugins/lisk-framework-forger-plugin/src/forger_plugin.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { Server } from 'http';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import * as os from 'os';
+import { KVStore } from '@liskhq/lisk-db';
+import {
+	ActionsDefinition,
+	BasePlugin,
+	BaseChannel,
+	EventsArray,
+	PluginInfo,
+} from 'lisk-framework';
+import { objects } from '@liskhq/lisk-utils';
+import * as express from 'express';
+import type { Express } from 'express';
+import * as cors from 'cors';
+import * as rateLimit from 'express-rate-limit';
+import * as controllers from './controllers';
+import * as middlewares from './middlewares';
+import * as config from './defaults';
+import { Options } from './types';
+
+// eslint-disable-next-line
+const pJSON = require('../package.json');
```

Comment:
I would just use packageJSON even though file might be small but ok to keep if you think it's readable as is.